crates/goose-server/src/routes/schedule.rs

```diff
@@ -264,9 +254,14 @@ async fn run_now_handler(
                         .ok()
                         .map(|(r, _)| r.version)
                     })
-            })
-            .unwrap_or_else(|| "unknown".to_string()),
-        Err(_) => "unknown".to_string(),
+                    .unwrap_or_else(|| "unknown".to_string());
+
+                (recipe_display_name, recipe_version)
+            } else {
+                (id.clone(), "unknown".to_string())
```

Comment:
is this really what you want to do? list the thing as unknown when we can't find it? I think doing a filter here would be better

Reply:
I don't want to return null's here

Reply:
well, if you can't find the thing here, what's the point of passing on id to the next thing? it won't be able to run the scheduler, will it? so I think returning null here and then just abort with an error of not found or some such is probably the right thing to do?